files/build_templates/docker_image_ctl.j2

```diff
@@ -62,6 +62,9 @@ start() {
         -v /usr/share/sonic/device/$PLATFORM:/usr/share/sonic/platform:ro \
 {%- if docker_container_name != "database" %}
         -v /usr/share/sonic/device/$PLATFORM/$HWSKU:/usr/share/sonic/hwsku:ro \
+{%- endif %}
+{%- if docker_container_name === "snmp" %}
+        -v /usr/lib/python3/dist-packages/arista:/usr/lib/python3/dist-packages/arista:ro \
```

Comment:
what if the arista directory does not exist?

Reply:
it will create an empty folder on the base image and mount it into the docker.

Reply:
instead of change this template, it is better to add RUN_OPT in snmp docker.

$(DOCKER_SNMP_SV2)_RUN_OPT += -v /usr/lib/python3/dist-packages/arista:/usr/lib/python3/dist-packages/arista:ro

can you try and test if that works or not. 

Also can you add comment why we need this.